File: lib/bench.rb
class Bench
	def self.bench_start()
		@@t0 = Time.now
	end

	def self.bench_end()
		@@t1 = Time.now
		res = ((@@t1.to_f - @@t0.to_f) * 1000.0).to_i
		puts res
	end
end

File: lib/bench.py
from time import clock

class Bench:
	@classmethod
	def bench_start(self):
		self.t0 = clock()

	@classmethod
	def bench_end(self):
		self.t1 = clock()
		delta = (self.t1 - self.t0)
		res = int(delta * 1000.0)
		print(res)